Lombiq.Analyzers/Build.props

```diff
@@ -2,21 +2,21 @@
   <PropertyGroup>
     <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)orchardcore.ruleset</CodeAnalysisRuleSet>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <!-- Don't run analyzers during Visual Studio build, not to slow down development. They still run during dotnet build.
-         By making this conditional, it won't overwrite the value if you set it in the Directory.Build.props file. For
-         Rider users: https://youtrack.jetbrains.com/issue/RIDER-80579 -->
     <RunAnalyzersDuringBuild Condition="'$(RunAnalyzersDuringBuild)' != 'true'">false</RunAnalyzersDuringBuild>
+    <!-- Don't run analyzers during Visual Studio build, not to slow down development. They still run during dotnet
+         build. By making this conditional, it won't overwrite the value if you set it in the Directory.Build.props
+         file. For Rider users: https://youtrack.jetbrains.com/issue/RIDER-80579 -->
```

Comment:
Why has this comment moved below the respective property?

Reply:
It shouldn't, a mistake.